### ISSUE TITLE
Report missing after training

### ISSUE THREAD

**Issue by kabikaj** [mittagessen/kraken#470]
Hi,
I am trying to make a test for training the recognizer with several exported page xml documents done in escriptorium. The following command produces a list of models apparently without errors but no report on the accuracy:

$ ketos -vv train d cuda:0 --batch-size 16 -o out/model -f page export_page_format/*.xml

If I used the best model against an unseen image, I don't get any results. I guess that that must be because I need to produce more training data, but I don't understand why I don't get a report from the training. I have the feeling that I am doing something wrong

thanks!
Alicia

**Comment by colibrisson** [comment]
Could you post the output you get?

**Comment by kabikaj** [comment]
sorry for the late reply. I tried to upgrade pytorch and ketos after posting my comment and I then I started training again. I waited until it finished and now it works!, so I guess the problem was related with the versions.
